### DISCUSSION TITLE
c#使用Avalonia，在麒麟操作系统下（linux）打开预览pdf如何实现？

### DISCUSSION THREAD

**Discussion by RRRRossi** [BYJRK/DotNet-Discussions#289] [其他框架]
在window环境下，引用PdfiumViewer.dll和pdfium.dll，实现了将本地指定路径下的pdf在程序中预览，请问在麒麟系统下，有哪位大佬实现了，在程序中预览pdf文档呢，谢谢

**Comment by klousCan** [comment]
https://github.com/arklumpus/MuPDFCore

**Reply by tenglongwentian** [reply]
webview+pdf.js传入文件绝对路径即可，高度定制化。而且免费。
